mscore/masterpalette.cpp

```diff
@@ -183,6 +184,7 @@ MasterPalette::MasterPalette(QWidget* parent)
 
       symbolItem = new QTreeWidgetItem();
       symbolItem->setData(0, Qt::UserRole, -1);
+      symbolItem->setText(0, QT_TRANSLATE_NOOP("MasterPalette", "Symbols"));
```

Comment:
This should fix the issue with getting "Symbols" translated and keep the shortcut working.